SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularitySchedulerPoller.java

```diff
@@ -72,4 +72,9 @@ public void runActionOnPoll() {
       "SingularitySchedulerPoller"
     );
   }
+
+  @Override
+  protected boolean abortsOnError() {
+    return false;
```

Comment:
why set this to `false`?

Reply:
To avoid killing the entire leader if the scheduler poller runs into an unexpected exception during a poll - currently it aborts at https://github.com/HubSpot/Singularity/blob/1415dea99de62cd0e39f4b3196d74b1feff5e6c7/SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityLeaderOnlyPoller.java#L150